src/main/steps/privacy-policy/content.ts

```diff
@@ -1,167 +1,213 @@
-  region: 'the region or town where you are using your computer, phone or tablet',
-  webBrowser: 'the web browser you use',
+  weUseCookies:
+    "We use <a class='govuk-link' href='/cookies'>cookies</a> to collect data that tells us about how you’re using this service, including:",
+  ifYouOpenEmail: 'if you open an email from us or click on a link in an email',
+  yourComputer: 'your computer, phone or tablet’s IP address',
+  theRegion: 'the region or town where you’re using your computer, phone or tablet',
+  theWeb: 'the web browser you use',
   storingYourData: 'Storing your data',
-  amountOfTime:
-    "When you make an application we store the data you have provided. The amount of time that your data is kept for depends on what you're applying for.",
+  postSubmissionData:
+    'After you submit your application, the information you entered will be stored for 50 years after the last update on your appeal and then deleted.',
   sharingYourData: 'Sharing your data',
-  anotherGovDepartment:
+  ifAnother:
     'While processing your application, another government department, agency or organisation might be involved and we may share your data with them.',
-  preventCrime:
-    'In some circumstances we may share your data, for example to prevent or detect crime, or to produce anonymised statistics.',
-  googleAnalytics:
-    'We use Google Analytics to collect data about how a website is used. This anonymous data is shared with Google. Find out about this in our terms an conditions.',
-  storingAndSharing: 'Storing and sharing your data internationally',
-  internationally:
+  ifYouContactUs:
+    'If you contact us and ask for help with the service you’re using, your personal data may be shared with the <a class="govuk-link" href="https://www.goodthingsfoundation.org/" target="_blank">Good Things Foundation (opens in a new window)</a>. This is a company who we have partnered with to offer face to face support.',
+  crime:
+    'In some circumstances we may share your data. For example, to prevent or detect crime, or to produce anonymised statistics.',
+  weUse:
+    "We use Google Analytics to collect data about how a website is used. This anonymous data is shared with Google. Find out about this in our <a class='govuk-link' href='/terms-and-conditions'>terms and conditions</a>.",
+  lawfulBases:
+    'Under the UK General Data Protection Regulation (UK GDPR), the lawful bases we rely on for processing this information are:',
+  consent:
+    'Your consent. You are able to withdraw your consent at any time. You can do this by contacting Disclosure Team, Post point 10.24, 102 Petty France, London, SW1H 9AJ',
+  legalObligation: 'We have a legal obligation.',
+  vitalInterest: 'We have a vital interest.',
+  publicTask: 'We need it to perform a public task.',
+  legitimateInterest: 'We have a legitimate interest.',
+  storeAndShare: 'Storing and sharing your data internationally.',
```

Comment:
storeAndShare shouldn't have dot at the end as this is a heading in bold. Same applies for welsh.